CorsixTH/Lua/graphics.lua

```diff
@@ -120,18 +120,25 @@ end
 --! Tries to load the font file given in the config file as unicode_font.
 --! If it is not found it tries to find one in the operating system.
 function Graphics:loadFontFile()
+  local lfs = require("lfs")
+  local function check(path) return path and lfs.attributes(path, "mode") == "file" end
   -- Load the Unicode font, if there is one specified.
-  local font_file = self.app.config.unicode_font
-  if not font_file then
-    -- Try a font which commonly comes with the operating system.
-    local windir = os.getenv("WINDIR")
-    if windir and windir ~= "" then
-      font_file = windir .. pathsep .. "Fonts" .. pathsep .. "ARIALUNI.TTF"
-    elseif self.app.os == "macos" then
-      font_file = "/Library/Fonts/Arial Unicode.ttf"
-    else
-      font_file = "/usr/share/fonts/truetype/arphic/uming.ttc"
-    end
+  local config_path = self.app.config.unicode_font
+  -- Try a font which commonly comes with the operating system.
+  local os_path, font_file
+  local windir = os.getenv("WINDIR")
+  if windir and windir ~= "" then
+    os_path = windir .. pathsep .. "Fonts" .. pathsep .. "ARIALUNI.TTF"
+  elseif self.app.os == "macos" then
+    os_path = "/Library/Fonts/Arial Unicode.ttf"
+  else
```

Comment:
Maybe be more exclusive here? We're assuming only Linux is left (even though it probably is)

Reply:
We generally assume that it'll follow the Filesystem Hierarchy Standard, and it's left for other platform developers to add something.